uw_msca/models.py

```diff
@@ -60,3 +60,15 @@ def json_data(self):
 
     def __str__(self):
         return json.dumps(self.json_data())
+
+
+class GoogleDriveState(models.Model):
+    # max_length values informed by examining all current results
+    # TODO: more thorough solution
+    drive_id = models.SlugField(max_length=19)
+    drive_name = models.SlugField(max_length=125)
```

Comment:
I need to check with Ken on the maximum drive name length but was wondering if [django's unbounded TextField was also supported](https://docs.djangoproject.com/en/5.0/ref/models/fields/#django.db.models.TextField) in case that was easier than trying to match whatever Google's rules were